decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
Whatever we decide I think we are reasonably safe since this patch includes tests for the most common 404 errors that switch to production mode on the fly and check that the error page is rendered properly. See https://github.com/decidim/decidim/blob/e762fb27e1899af59e094283334b0b3676068b07/decidim-core/lib/decidim/core/test/shared_examples/errors.rb.